utils/debug/DebugUvMaterial.js

```diff
@@ -0,0 +1,13 @@
+import * as THREE from 'three';
+import debugUV_VS from './shader/debugUV_VS.glsl';
+import debugUV_FS from './shader/debugUV_FS.glsl';
+
+const debugUvMaterial = new THREE.ShaderMaterial({
```

Comment:
In fact I really don't think we need this: even three doesn't have this. So I fail to see why as a GIS framework we should add a debugging shader.

If this is useful to you: keep it in a branch / stash / patch, but not in itowns repository.

Reply:
I think it can be usefull in the for dev that will deals with image/texture in the future, like projective texture. And I think it's not very usable for other dev if it's hidden in a branch...

I can add a checkbox in my example to show the debug shader, so it has more sense to keep it.
What do you think about this compromise ?

Reply:
+1 for keeping the uvMaterial and the cameraHelper in this PR, but I think we need a gui option (like in a Debug or Advanced subfolder) in the example to activate/deactivate them (they are currently [commented out](https://github.com/iTowns/itowns/blob/14c18b31122c44e150e0a2581b5fb9f7208d1301/examples/oriented_images.html#L138-L142)).

Reply:
You only have to add one line in any shader to display the uv so I really don't see the added value of this `DebugUvMaterial.js` :-(

Reply:
No strong opinion, but it may be very handy to have this material already setup for itowns (logdepth support...). Do you feel it is outweighted by the maintenance burden of these few lines of code in debug ?